getting_started/intel_debug/introduction.rst

```diff
@@ -160,41 +168,46 @@ Notes:
+|Tiger Lake and newer (community signed)                    |        |/lib/firmware/intel/sof-ipc4/PLAT/community/    |           |                                   |
++-----------------------------------------------------------+--------+------------------------------------------------+-----------+-----------------------------------+
+|Meteor Lake and newer Loadable libraries                   |IPC4    |/lib/firmware/intel/sof-ipc4-lib/PLAT/          |UUID.bin   |PLAT = mtl, lnl, ...               |
++-----------------------------------------------------------+        +------------------------------------------------+           |                                   |
+|Meteor Lake and newer Loadable libraries (community signed)|        |/lib/firmware/intel/sof-ipc4-lib/PLAT/community/|           |                                   |
++-----------------------------------------------------------+--------+------------------------------------------------+-----------+-----------------------------------+
+
+Look-up paths per Intel platform for **modular releases (IPC4 only)**
+
+.. _intel_modular_firmware_paths:
+
++------------------------------------------------------------+------------------------------------------------+-------------------------+---------------+
+|Platform                                                    |Load path                                       |File name                |Notes          |
++============================================================+================================================+=========================+===============+
+|Panther Lake and newer                                      |/lib/firmware/intel/sof-ipc4/PLAT/              ||                        |PLAT = ptl, ...|
+|                                                            |                                                || sof-PLAT.ri            |               |
+|                                                            |                                                || sof-PLAT-openmodules.ri|               |
+|                                                            |                                                || sof-PLAT-debug.ri      |               |
++------------------------------------------------------------+------------------------------------------------+                         |            -- |
+|Panther Lake and newer (community signed)                   |/lib/firmware/intel/sof-ipc4/PLAT/community/    |                         |               |
++------------------------------------------------------------+------------------------------------------------+-------------------------+            -- |
+|Panther Lake and newer Loadable libraries                   |/lib/firmware/intel/sof-ipc4-lib/PLAT/          |UUID.bin                 |               |
++------------------------------------------------------------+------------------------------------------------+                         |            -- |
```

Comment:
@ujfalusi Also, why not just put UUID.bins to the main directory? The UUID namespace is unique by definition, so what do we gain by putting these to sof-ipc4-lib versus sof-ipc4?

Reply:
I cannot find the thread on why we ended up with a separate directory for the libraries, it is mentioned here at least: https://github.com/thesofproject/linux/pull/3826#discussion_r949059248

I think it has something to do with the fact that libraries can be signed with a vendor key and in that case they cannot be alongside of the Intel or community signed basefw, but the basefw can be only signed with Intel or community key.

But this has been the rule ever since we have the UUID based library lookup. Initially I wanted to toss them alongside of the basefw, but then we have changed direction and there were a good reason to do that (but what? I cannot recall).

Reply:
Ack, thanks @ujfalusi . So this goes to "water under the bridge" category, let's stick to what we've done already and be consistent with previous IPC4 FW releases.